app/src/main/java/com/ustwo/boilerplate/base/BasePresenter.java

```diff
@@ -41,7 +40,6 @@ public void onViewWillShow(@NonNull final T view) {
      */
     @CallSuper
     public void onViewWillHide() {
```

Comment:
For me usage of `visibleDisposables` with `onViewWillShow()` and `onViewWillHide()` methods on are a bit more like a rare case and I'd move it to the example keeping in mind to move them back as soon as we realise they definitely should be a part of `BasePresenter` (Boilerplate doesn't have to be rigid, does it?). But I'd rely on your experience @mshipton.